content/handbook/engineering/ai/context-systems/_index.md

```
@@ -46,7 +46,11 @@ GitLab の Issue／MR でチーム全体に連絡を取るには `@gitlab-org/co
 
 | カテゴリ          | カウンターパート                                                                       |
 |-------------------|--------------------------------------------------------------------------------------|
-| プロダクトマネージャー   | <p class="my-3 text-sm text-gray-600 italic">チームメンバー情報は <a href="https://handbook.gitlab.com/handbook/engineering/ai/context-systems/#team-members" rel="external noopener">原文 (英語)</a> を参照してください。</p> |
-| テクニカルライティング | <p class="my-3 text-sm text-gray-600 italic">チームメンバー情報は <a href="https://handbook.gitlab.com/handbook/engineering/ai/context-systems/#team-members" rel="external noopener">原文 (英語)</a> を参照してください。</p> |
+| プロダクトマネージャー   | 
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)